common/djangoapps/enrollment/handlers.py

```diff
@@ -0,0 +1,67 @@
+"""
+Handlers and actions related to enrollment.
+"""
+import logging
+
+from edxmako.shortcuts import render_to_string
+from django.conf import settings
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+from django.core.urlresolvers import reverse
+from django.core.mail import send_mail
+from django.dispatch import receiver
+
+from student.models import ENROLL_STATUS_CHANGE, EnrollStatusChange
+
+log = logging.getLogger(__name__)
+
+
+@receiver(ENROLL_STATUS_CHANGE)
+def send_email_to_staff_on_student_enrollment(sender, event=None, user=None, **kwargs):  # pylint: disable=unused-argument
+    """
+    Sends an e-mail to staff after a new enrollment.
+    This feature can be enabled by setting the e-mail in ENROLLMENT_NOTIFICATION_EMAIL in lms.env.json, or by using
+    a SiteConfiguration variable of the same name. Disabled by default.
+    """
+
+    if event == EnrollStatusChange.enroll:
+        # If there's a SiteConfiguration variable (including ''), we use it instead of the lms.env.json setting
+        to_email = configuration_helpers.get_value('ENROLLMENT_NOTIFICATION_EMAIL',
+                                                   settings.ENROLLMENT_NOTIFICATION_EMAIL)
+
+        if not to_email:
+            # feature disabled
+            return
+
+        course_id = kwargs['course_id']
+
+        site_protocol = 'https' if settings.HTTPS == 'on' else 'http'
+        site_domain = configuration_helpers.get_value('site_domain', settings.SITE_NAME)
+        course_url = u"{protocol}://{domain}{path}".format(
+            protocol=site_protocol,
+            domain=site_domain,
+            path=reverse('about_course', args=[course_id.to_deprecated_string()])
+        )
+        user_admin_url = u"{protocol}://{domain}{path}".format(
+            protocol=site_protocol,
+            domain=site_domain,
+            path=reverse('admin:auth_user_change', args=[user.id]),
+        )
+
+        subject = render_to_string('emails/new_enrollment_email_subject.txt', {})
+        message = render_to_string('emails/new_enrollment_email_body.txt', {
+            'user': user,
+            'course_url': course_url,
+            'user_admin_url': user_admin_url,
+        })
+
+        try:
+            send_mail(
```

Comment:
Nit: can we use `django.core.mail.message.EmailMessage` (example: https://github.com/edx/edx-platform/pull/16624/files#diff-5a31e91b78b2599dc5ace9c2a3b31d8cR776)? It may allow a little more flexibility later.